loom/common/core/allocator.h

```diff
@@ -22,6 +22,9 @@
 #define _CORE_ALLOCATOR_H_
 
 #include "loom/common/core/assert.h"
+#include "jemalloc/jemalloc.h"
+#include <stdint.h>
```

Comment:
We should use the standard Loom platform headers we use for those things.
If our platform.h doesn't give it to us we should include there.

On Tue, Dec 22, 2015 at 3:46 PM, Miha Lunar <notifications@github.com>
wrote:

> In loom/common/core/allocator.h
> <https://github.com/LoomSDK/LoomSDK/pull/442#discussion_r48311182>:
>
> > @@ -22,6 +22,9 @@
> >  #define _CORE_ALLOCATOR_H_
> >
> >  #include "loom/common/core/assert.h"
> > +#include "jemalloc/jemalloc.h"
> > +#include <stdint.h>
>
> Not 100% on cross-platformness, but they're used for bool and e.g.
> uint32_t. I should move them to allocator.c anyway, since header file
> doesn't use them anymore. In any case, if we run into trouble we can
> include LuaJIT's compat version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LoomSDK/LoomSDK/pull/442/files#r48311182>.
>